Formula/pytouhou.rb

```diff
@@ -25,7 +25,7 @@ class Pytouhou < Formula
   depends_on "gtk+3" => :recommended
   if build.with? "gtk+3"
     depends_on "py3cairo" # FIXME: didn't get picked up by pygobject3 below
-    depends_on "pygobject3" => "with-python3"
```

Comment:
It makes us pass the audit, but it will surely disable functionality. We need to think more about what to do here